src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -227,13 +227,41 @@ class LazyClassLoaderHelper
             });
     }
 
+    llvm::Value* getVtblOffset(llvm::IRBuilder<>& builder, llvm::Twine fieldDescriptor, llvm::StringRef methodName,
+                               llvm::StringRef typeDescriptor)
+    {
+        return returnConstantForClassObject(builder, fieldDescriptor, methodName + ";" + typeDescriptor,
+                                            [=](const ClassObject* classObject)
+                                            {
+                                                const Method* iter;
+                                                do
+                                                {
+                                                    llvm::ArrayRef<Method> methods = classObject->getMethods();
+                                                   iter =
+                                                        llvm::find_if(methods,
+                                                                      [&](const Method& method) {
+                                                                          return !method.isStatic()
+                                                                                 && method.getName() == methodName
+                                                                                 && method.getType() == typeDescriptor;
+                                                                      });
+                                                   if (iter != classObject->getMethods().end())
```

Comment:
You can reuse `methods` here. So `methods.end()` instead of `classObject->getMethods().end()`